ntbs-service/wwwroot/source/Components/ValidateRiskFactor.ts

```diff
@@ -0,0 +1,62 @@
+import Vue from 'vue';
+const axios = require('axios');
+
+type RiskFactor = { 
+  inPastFiveYearsCheckbox: any, 
+  moreThanFiveYearsAgoCheckbox: any,
+  isCurrentCheckbox: any,
+  status: any
+}
+
+const ValidateRiskFactor = Vue.extend({
+  props: ['model', 'property'],
+  data: function() {
+    return {
+      hasError: false
+    }
+  },
+  methods: {
+    validate: function (event: FocusEvent) {
+        // Do nothing if focused element is part of the div that lost focus
+        if (this.$el.contains(event.relatedTarget)) {
+            return;
+        }
+        
+        var headers = {
+            "RequestVerificationToken": (<HTMLInputElement>document.querySelector('[name="__RequestVerificationToken"]')).value
+        }
+
+        var riskFactor = this.getRiskFactor();
+        const url = `${this.$props.model}/Validate${this.$props.model}Property?key=${this.$props.property}&pastFive=${riskFactor.inPastFiveYearsCheckbox}&moreThanFive=${riskFactor.moreThanFiveYearsAgoCheckbox}&isCurrent=${riskFactor.isCurrentCheckbox}&status=${riskFactor.status}`;

```

Comment:
This is getting quite unwiedly..
Can we provide the parameters through a config object?
https://github.com/axios/axios#request-config